### ISSUE TITLE
Sidebar ul,li is a bit of a mess

### ISSUE THREAD

**Issue by ToeiRei** [BlackrockDigital/startbootstrap-sb-admin-2#246]
I am currently trying to use the template in a laravel project, but the menu structure is a bit messy for working with menu generation.

Dividers and the branding don't seem to care if you add `<li></li>` around them, but the sidebar toggler shifts, which looks ugly.

Submenu implementation is pretty hard due to the `<div>` implementation as common menu generation would expect another set of `<ul></ul>` in there...

**Comment by davidtmiller** [comment]
Agreed, this will be changed out in a future build with something a bit more streamlined.
